OverlayPlugin.Core/resources/opcodes.jsonc

```diff
@@ -12,8 +12,6 @@
         }
     },
     "2022.07.05.0000.0000": {
-        // Region: KR
-        // Version: 6.05h
```

Comment:
Should restore these two lines, or delete the entire `2022.07.05.0000.0000` entry.